decision-tree.yaml

```
@@ -546,32 +518,61 @@ questions:
         url: https://eur-lex.europa.eu/legal-content/NL/TXT/HTML/?uri=OJ:L_202401689#rct_53
     answers:
       - answer: Ja
+        nextQuestionId: "2.7" #profilering
+      - answer: Nee
         labels:
-          - "geen hoog-risico AI"
+          - "hoog-risico AI"
         redirects:
-         - nextQuestionId: "2.9" #transparantieverplichting
+         - nextQuestionId: "2.7" #2.7 = profilering, 2.4.2 = conformiteitsbeoordeling derde partij
```

```suggestion
         - nextQuestionId: "2.7" #2.7 = profilering
```
2.4.2 here is not relevant for this question right? So why even put it in the comments?

---

Yes I agree!